test fail looks real

======================================================================
ERROR: test_backend (corehq.apps.sms.tests.BackendTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/sms/tests/__init__.py", line 192, in tearDown
    self.backend8.delete()
  File "/home/travis/virtualenv/local/lib/python2.7/site-packages/couchdbkit/schema/base.py", line 202, in delete
    raise TypeError("the document is not saved")
TypeError: the document is not saved
----------------------------------------------------------------------

--- a/corehq/apps/sms/tests/__init__.py
+++ b/corehq/apps/sms/tests/__init__.py
@@ -189,10 +189,6 @@ class BackendTestCase(BaseAccountingTest):
         self.backend7.delete()
         self.backend_mapping7.delete()
 
-        self.backend8.delete()
-        self.backend9.delete()
-        self.backend10.delete()
-
         self.contact.delete_verified_number()
         self.case.delete()
 

